spec/lib/data_cycle/daily_update_spec.rb

```diff
@@ -14,20 +14,39 @@
 
   describe 'on initialization' do
     it 'calls lots of update routines' do
-      expect(UserImporter).to receive(:update_users)
-      expect(AssignedArticleImporter).to receive(:import_articles_for_assignments)
-      expect(ArticlesCoursesCleaner).to receive(:rebuild_articles_courses)
-      expect(RatingImporter).to receive(:update_all_ratings)
-      expect(ArticleStatusManager).to receive(:update_article_status)
-      expect(OresScoresBeforeAndAfterImporter).to receive(:import_all)
-      expect(UploadImporter).to receive(:find_deleted_files)
-      expect_any_instance_of(OverdueTrainingAlertManager).to receive(:create_alerts)
-      expect(PushCourseToSalesforce).to receive(:new)
-      expect(UpdateCourseFromSalesforce).to receive(:new)
-      expect(Raven).to receive(:capture_message).and_call_original
-      update = described_class.new
-      sentry_logs = update.instance_variable_get(:@sentry_logs)
-      expect(sentry_logs.grep(/Pushing course data to Salesforce/).any?).to eq(true)
+      VCR.use_cassette 'revision_scores/deleted_revision' do
```

Comment:
It's good to test for the behavior of revision score importer, but the `DailyUpdate` spec is not the right place to do that.

Ideally, DailyUpdate's own spec is just testing the behavior of `DailyUpdate` itself, and won't break because of implementation details of `RevisionScoreImporter`... and `RevisionScoreImporter`'s specs should break when its behavior changes, rather than relying on a test of a different class to check some of its behavior.